internal/providers/github/webhook/handlers_repos.go

```diff
@@ -132,13 +131,10 @@ func sendEvaluateRepoMessage(
 	repo *repo,
 	handler string,
 ) (*processingResult, error) {
-	lookByProps, err := properties.NewProperties(map[string]any{
+	lookByProps := properties.NewProperties(map[string]any{
```

Comment:
I don't know how much of a pain you want me to be... this method no longer returns a non-`nil` error, so we _could_ remove that from the signature.  On the other hand, I like more-focused changes, so maybe hold off?